addons/sys_sem70/CfgWeapons.hpp

```diff
@@ -26,7 +26,7 @@ class CfgWeapons {
         type = 4096;
         simulation = "ItemMineDetector";
         class ItemInfo {
-            mass = 120;
+            mass = 176.2;
```

Comment:
Just pointing out that these are supposed to be integers.

Reply:
I dont know, They work, that is what I know.